backend/pkg/httpserver/server_test.go

```diff
@@ -388,22 +389,24 @@ func (m *MockWPTMetricsStorer) GetFeature(
 
 func (m *MockWPTMetricsStorer) ListBrowserFeatureCountMetric(
 	_ context.Context,
-	browser string,
+	targetBrowser string,
+	targetMobileBrowser *string,
 	startAt time.Time,
 	endAt time.Time,
 	pageSize int,
 	pageToken *string,
 ) (*backend.BrowserReleaseFeatureMetricsPage, error) {
 	m.callCountListBrowserFeatureCountMetric++
 
-	if browser != m.listBrowserFeatureCountMetricCfg.expectedBrowser ||
+	if targetBrowser != m.listBrowserFeatureCountMetricCfg.expectedTargetBrowser ||
+		targetMobileBrowser != &m.listBrowserFeatureCountMetricCfg.expectedTargetMobileBrowser ||
```

Comment:
This is only comparing the address. You should compare with reflect.DeepEqual like with the pageToken
```suggestion
		targetMobileBrowser != &m.listBrowserFeatureCountMetricCfg.expectedTargetMobileBrowser ||
```